src/server/wesnothd/game.hpp

```diff
@@ -614,13 +621,24 @@ class game
 		observers_.clear();
 	}
 
-	bool is_queue_game() const
+	queue_type::type q_type() const
 	{
-		return is_queue_game_;
+		return queue_type_;
 	}
-	void is_queue_game(bool is_queue_game)
+	void q_type(queue_type::type queue_type)
```

Comment:
This does not seem used and changing the queue type of an existing game does not sound not like a good idea.

Reply:
Updated.